presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeGeneralQueries.java

```diff
@@ -69,4 +69,7 @@ public void testTopN() {}
     @Override
     @Ignore
     public void testInsertIntoSpecialPartitionName(){}
+    @Override
```

Comment:
Nit : Please add a newline between the tests.

Reply:
+1